sulong/projects/com.oracle.truffle.llvm.runtime/src/com/oracle/truffle/llvm/runtime/LLVMContext.java

```diff
@@ -87,6 +90,16 @@
 import com.oracle.truffle.llvm.runtime.types.Type;
 
 public final class LLVMContext {
+    // my code start
+    public final ConcurrentMap<LLVMPointer, Object> condStorage;
+    public final ConcurrentMap<LLVMPointer, Object> mutexStorage;
+    public final ConcurrentMap<LLVMPointer, Object> rwlockStorage;
+    public final ConcurrentMap<Long, Object> retValStorage;
+    public final ConcurrentMap<Long, Thread> threadStorage;
```

Comment:
Maybe add a comment here that the `Long` is the thread ID.

Reply:
added // the long-key is the thread-id